sdk/cosmos/azure_data_cosmos/src/options/mod.rs

```diff
@@ -43,10 +50,71 @@ pub struct DeleteDatabaseOptions<'a> {
     pub method_options: ClientMethodOptions<'a>,
 }
 
+/// Options to be passed to ['ItemOptions'](crate::options::ItemOptions).
```

Comment:
We should rephrase this to something like `Specifies consistency levels that can be used when working with Cosmos APIs`. And include a link to the docs: https://learn.microsoft.com/en-us/azure/cosmos-db/consistency-levels